### DISCUSSION TITLE
Cannot connect to microsoft sql from Cypress

### DISCUSSION THREAD

**Discussion by shivanijamadar** [cypress-io/cypress#16970] [Questions and Help]
Hello @jennifer-shehane below is my code to access database but getting the following error, can you please guide?

Error:
![image](https://user-images.githubusercontent.com/57283203/122348962-5769cf80-cf69-11eb-9fc1-12d17e897efc.png)

index.ts:

const cucumber = require('cypress-cucumber-preprocessor').default
const mssql = require("mssql");
const syncSql = require("sync-sql");

module.exports = (on, config) => {
  
  on('file:preprocessor', cucumber())
  on('task', {
    getDBDataSync : getDBDataSync
    
  });
  return config;
}

function getDBDataSync(){
  const config = {
        host : "192.168.44.136\\qa" ,
        port: 1433,
        Login : "Test1",
        Pwd : "Test1",
        database: "master",
        
}
  const query = "select * from dbo.spt_fallback_db";
  return syncSql.mssql(config, query);
} 

spec.js :
cy.task("getDBDataSync").then((result) => {
  console.log(result);
 } );

**Comment by Sanzhanov** [comment]
Hi @shivanijamadar, did you resolve this issue?
